tests/Hosting.Services.Web.UnitTests/Middlewares/UserIdentity/HttpContextHelper.cs

```diff
@@ -17,15 +19,26 @@ public static HttpContext GetContextWithIp(string address)
 			return context;
 		}
 
+		public static HttpContext GetContextWithEmail(string email)
+		{
+			(HttpContext context, _) = CreateHttpContext();
+			byte[] emailBytes = System.Text.Encoding.UTF8.GetBytes($"{{\"Email\":\"{email}\"}}");
+			context.Request.Body.Write(emailBytes, 0, emailBytes.Length);
+			return context;
+		}
+
 		public static (HttpContext context, HttpConnectionFeature feature) CreateHttpContext()
 		{
 			HttpConnectionFeature feature = new();
 
 			FeatureCollection features = new();
 			features.Set<IHttpConnectionFeature>(feature);
 
+			Stream requestBody = new MemoryStream();
```

Comment:
Add a using before this line, to ensure it is properly disposed

Reply:
Not able to add using as we need to write and read from this stream, manually closing them when we use `GetContextWithEmail()`